app/controllers/albums_controller.rb

```diff
@@ -6,10 +6,26 @@ def index
     @albums = Album.all
   end
 
+  def user_albums
+    @albums = current_user.albums
+  end
+
   # GET /albums/1 or /albums/1.json
   def show
   end
 
+  def add_album_to_playlist
+    @user_album = UserAlbum.new(user_id: current_user.id, album_id: params[:id])
+    @user_album.save!
+  end
```

Comment:
this method for UserAlbum#create
```suggestion
  def add_album_to_playlist
    @user_album = UserAlbum.new(user_id: current_user.id, album_id: params[:id])
    @user_album.save!
  end
```